proto/brpb.proto

```diff
@@ -898,6 +898,48 @@ message MetaEdit {
     bool destruct_self = 4;
 }
 
+// RewrittenTableID records a sort of modification over the SSTs during restoring.
+// 
+// When "backing up" this "restored" SST, we want to backup what it was "restored".
```

Comment:
I feel like `we want to back up the original SST before any rewrite` maybe more intuitive?